src/main/kotlin/no/nav/tilleggsstonader/sak/statistikk/behandling/BehandlingsstatistikkService.kt

```diff
@@ -194,8 +194,8 @@ class BehandlingsstatistikkService(
             relatertBehandlingId = relatertBehandlingId,
             versjon = Applikasjonsversjon.versjon,
             vilkårsprøving = emptyList(),
-            revurderingÅrsak = null,
-            revurderingOpplysningskilde = null,
+            revurderingÅrsak = saksbehandling.nyeOpplysningerMetadata?.endringer?.joinToString(","),
```

Comment:
Innspill på denne? Vi gjorde slik da vi var usikker på om det var noe i veien for å endre typen til en liste? (er en nullable String nå)